apps/meteor/client/lib/chats/flows/uploadFiles.ts

```diff
@@ -61,7 +63,11 @@ export const uploadFiles = async (chat: ChatAPI, files: readonly File[], resetFi
 						return;
 					}
 
-					const shouldConvertSentMessages = e2eRoom.shouldConvertSentMessages({ msg });
+					const subscription = await waitUntilFind(() => ChatRoom.findOne({ _id: room._id }));
```

Comment:
The `room` object declared on line 19 should be the same as this. Is it possible to use it instead?

Reply:
Done!